ivy_tests/test_ivy/test_frontends/test_paddle/test_math.py

```diff
@@ -943,6 +943,42 @@ def test_paddle_floor_divide(
     )
 
 
+# floor_mod
```

Comment:
Tests are failing in CI, can you show me the screenshots of your test results locally? 😄

Reply:
> Tests are failing in CI, can you show me the screenshots of your test results locally? 😄

<img width="758" alt="image" src="https://github.com/unifyai/ivy/assets/82304628/143f5e82-d4cb-43ea-a31e-00c3b175a16d">